src/components/PostGrid.tsx

```diff
@@ -34,53 +38,89 @@ const PostGrid = ({ data, user }: PostGridProps): JSX.Element => {
   }
 
   return (
-    <div className="bg-offWhite rounded-xl pt-6 px-4 pb-2 font-roboto">
-      <div className="w-full flex justify-between items-center mb-3">
-        <div className="flex w-full">
-          {data?.contributions[0] &&
-            <Avatar
-              contributor={data.contributions[0]?.contributor}
-              lastPr={dayjs(data.contributions[0]?.last_merged_at).fromNow()}/>}
+    <div className="">
+      <div className='border-[1px] border-gray-100 rounded-[14px]'>
+        <div className='flex flex-col rounded-t-[14px] bg-gray-100 p-[30px]'>
+          <div className='w-full flex justify-end '>
+            <div
+              role="button"
+              tabIndex={0}
+              aria-pressed="false"
+              onClick={() => user_id ? handleVoteUpdateByRepo(votes, repo_id) : signIn({ provider: 'github' })}
+              onKeyDown={(e) => {
+                if (e.key === 'Enter') {
+                  return user_id ? handleVoteUpdateByRepo(votes, repo_id) : signIn({ provider: 'github' });
+                }
+              }}
+              className="flex justify-center items-center text-base space-x-1 bg-gray-50 py-[2px] px-[5px] rounded-[5px] cursor-pointer transition-all duration-200"
+            >
+              <img src="/UpvoteIcon.png" alt="" />
```

Comment:
Does this alt property have to be empty like that ?

Reply:
Yep! An alter text should be there :)